lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -152,6 +152,7 @@ ProtocolParameters
     protocolParametersTxMaxSize             Word16          sql=tx_max_size
     protocolParametersDecentralizationLevel Percentage      sql=decentralization_level
     protocolParametersDesiredNumberOfPools  Word16          sql=desired_pool_number
+    protocolParametersMinimumUtxoValue      W.Coin          sql=minimum_utxo_value
```

Comment:
We're going to need a migration for this one with a default value.

Reply:
cherry-picked one commit and also handled the migration in https://github.com/input-output-hk/cardano-wallet/pull/1894/commits/487bb4f92fe88eff3f38cd02107337e9735ac8c3